src/common/state/redis.cc

```diff
@@ -941,10 +943,11 @@ void redis_task_table_add_task(TableCallbackData *callback_data) {
 
   int status = redisAsyncCommand(
       context, redis_task_table_add_task_callback,
-      (void *) callback_data->timer_id, "RAY.TASK_TABLE_ADD %b %d %b %b %b",
+      (void *) callback_data->timer_id, "RAY.TASK_TABLE_ADD %b %d %b %b %b %d",
```

Comment:
It might be worth adding a line

```C++
int spillback_count = execution_spec->getSpillbackCount();
```

above and passing `spillback_count` into `redisAsyncCommand`.

The reason is that I've seen hard-to-debug issues when there is a slight mismatch with the integer types passed in to `redisAsyncCommand`. If the return type of `getSpillbackCount` changes (e.g., to `int64_t`) then that will cause subtle crashes here and it will be hard to debug.

Reply:
I see, I'll type cast it.

Reply:
in that case, it seems we should also type cast the SpecSize() as well, it actually returns int64_t.

Reply:
oh but that's not being passed into a `%d`, that's being passed as the size of a `%b`, which actually needs to be `size_t` or something like that.